azure-pipelines.yaml

```diff
@@ -210,8 +219,18 @@ jobs:
           filePath: scripts/jira-status.sh
           arguments: $(jira-username) $(jira-password)
       - task: Bash@3
-        displayName: 'Send slack message'
+        displayName: 'Send slack message - pull request'
+        condition: |
+          and(succeeded(), eq(variables['isMain'], false), eq(variables['isAutoTriggered'], true))
+        inputs:
+          targetType: filePath
+          filePath: scripts/send-slack-message-pr.sh
+          arguments: $(dtspo-daily-checks-slack-bot-token) ${{ variables.pull_request_number }} $(github-management-api-token)
+      - task: Bash@3
+        displayName: 'Send slack message - master branch'
+        condition: |
+          and(succeeded(), eq(variables['isMain'], true), eq(variables['isAutoTriggered'], true))
```

Comment:
if someone is running the build manually on main they probably want it to go to the main channel? not a big deal as it will just show up the next day, but what if there was a bug in the previous output and they want to retrigger it?

Reply:
Yeah, was just wondering this myself. Submitting an update so it can be triggered manually

Reply:
https://github.com/hmcts/dtspo-daily-monitoring/pull/38